test/integration/update_patient_info_test.rb

```diff
@@ -138,6 +132,45 @@ class UpdatePatientInfoTest < ActionDispatch::IntegrationTest
     end
   end
 
+  describe 'changing fulfillment information' do
+    before do
+      @user.update role: :admin
+      @patient = create :patient, appointment_date: 2.days.from_now,
+                                              clinic_name: 'Sample Clinic 1'
+      create :pregnancy, patient: @patient,
+                         dcaf_soft_pledge: 100,
+                         pledge_sent: true
+      create :fulfillment, patient: @patient
+      visit edit_patient_path @patient
+      # find('#submit-pledge-button').click
+      # assert false
+
+      click_link 'Pledge Fulfillment'
+      check 'Pledge fulfilled'
+      fill_in 'Procedure date', with: 2.days.from_now.strftime('%m/%d/%Y')
+      select '12 weeks', from: 'Weeks along at procedure'
+      fill_in 'Abortion care $', with: '100'
+      fill_in 'Check #', with: '444-22'
+      fill_in 'Procedure date', with: 2.weeks.from_now.strftime('%m/%d/%Y')
```

Comment:
In line 150 you set `Procedure date` to `2 days` from now, but then here on line 154 you set `Procedure date` to `2 weeks` from now. Did you mean to set two distinct variables or is this a duplicate?